### PR TITLE
Update version of 'webpack-dev-server'

### PR DESCRIPTION
**Pull request by evidolob**
Signed-off-by: Yevhen Vydolob <yvydolob@redhat.com>

### What does this PR do?
Update version of 'webpack-dev-server' to 3.1.11 to fix CVE-2018-14732 

### What issues does this PR fix or reference?
https://nvd.nist.gov/vuln/detail/CVE-2018-14732

### CODE REVIEW

File: workspace-loader/package.json [hunk 1]
```diff
@@ -23,7 +23,7 @@
     "typescript": "^2.7.1",
     "uglifyjs-webpack-plugin": "^1.1.8",
     "webpack": "^3.10.0",
-    "webpack-dev-server": "^2.11.1",
+    "webpack-dev-server": "^3.1.11",
```
Comment (benoitf) [inline workspace-loader/package.json:27]: dev server v3.x is requiring a webpack 4.0 no ?
```
"peerDependencies": {
    "webpack": "^4.0.0"
  },
```

here we're using webpack 3

Comment (evidolob) [reply workspace-loader/package.json:27]: Yes, it's correct, then I need to update webpack to 4.0 :(

**Comment by evidolob** [comment]
@benoitf I add migration to webpack 4, can you review again?

**Comment by evidolob** [comment]
ci-test

File: workspace-loader/webpack.dev.js [hunk 2]
```diff
@@ -35,18 +36,18 @@ module.exports = merge(common, {
         historyApiFallback: true,
         proxy: {
             '/api/websocket': {
-                target: 'http://localhost:8080',
+                target: 'http://che-che.10.101.162.245.nip.io.',
```
Comment (benoitf) [inline workspace-loader/webpack.dev.js]: looks like macOS settings vs linux ?

**Review by benoitf — APPROVED** [review]

File: workspace-loader/webpack.dev.js (hunk 2 above)
Comment (evidolob) [reply workspace-loader/webpack.dev.js]: No, docker vs k8's :)

**Comment by evidolob** [comment]
ci-test

**Review by musienko-maxim — APPROVED** [review]

**Comment by artaleks9** [comment]
Selenium tests execution on Eclipse Che Multiuser on OCP (https://ci.codenvycorp.com/job/che-pullrequests-test-ocp/1500/) doesn't show any regression against this Pull Request.

**Pull request merged**